lib/rspec/core/configuration.rb

```diff
@@ -1597,6 +1597,39 @@ def define_derived_metadata(*filters, &block)
         @derived_metadata_blocks.append(block, meta)
       end
 
+      # Defines a callback that runs after the first example with matching
+      # metadata is defined. If no examples are defined with matching metadata,
+      # it will not get called at all.
+      #
+      # This can be used as an alternative to `before(:suite)` to define some
```

Comment:
As `before(:suite)` doesn't support the metadata filtering I'd +1 a rewrite here